### ISSUE TITLE
client/package.json has problems...

### ISSUE THREAD

**Issue by bkmartinjr** [chanzuckerberg/cellxgene#146]
Two issues I noted:
1. there are now several packages listed as both dev and production dependencies.  Includes:
    * Jest
    * webpack-hot-middleware
    * whatwg-fetch
2. these duplications have different minimum version requirements (eg, Jest has both 23.0.1 and 23.4.1)

**Comment by bkmartinjr** [comment]
More info -- a lot of the `npm install` errors are due to this duplication (or more specifically, the fact that we install obsolete versions of some packages).

When I manually cleaned up these package.json misconfigs, most of the errors went away.

```npm WARN deprecated babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read babeljs.io/env to update!
npm WARN deprecated sw-precache@4.3.0: Please update to v5 or higher to resolve an issue with redirected navigation responses. See https://github.com/GoogleChrome/sw-precache/releases/tag/5.0.0
npm WARN deprecated istanbul-lib-hook@1.2.1: 1.2.0 should have been a major version bump
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN deck.gl@4.1.6 requires a peer of react@0.14.x - 15.x but none is installed. You must install peer dependencies yourself.
npm WARN deck.gl@4.1.6 requires a peer of react-dom@0.14.x - 15.x but none is installed. You must install peer dependencies yourself.
npm WARN halogen@0.2.0 requires a peer of react@^0.14.8 but none is installed. You must install peer dependencies yourself.
npm WARN halogen@0.2.0 requires a peer of react-dom@^0.14.8 but none is installed. You must install peer dependencies yourself.
npm WARN react-router-dom@4.1.2 requires a peer of react@^15 but none is installed. You must install peer dependencies yourself.
npm WARN enzyme@2.9.1 requires a peer of react@0.13.x || 0.14.x || ^15.0.0-0 || 15.x but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-jsx-a11y@3.0.2 requires a peer of eslint@^2.10.2 || 3.x but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-react@6.10.3 requires a peer of eslint@^2.0.0 || ^3.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN react-addons-test-utils@15.6.2 requires a peer of react-dom@^15.4.2 but none is installed. You must install peer dependencies yourself.
npm WARN postcss-modules-resolve-imports@1.3.0 requires a peer of postcss@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN cellxgene@0.0.1 No description
npm WARN The package jest is included as both a dev and production dependency.
npm WARN The package webpack-hot-middleware is included as both a dev and production dependency.
npm WARN The package whatwg-fetch is included as both a dev and production dependency.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
```

**Comment by bkmartinjr** [comment]
fixed with PR #151
